src/current/v24.3/data-resilience.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Data Resilience
```

Comment:
Did we decide on Data Resilience as the title? What about Resilience in CRDB or something or is that too marketing-y

Reply:
@alicia-l2 What do you think about trying out "Data Resilience" for now? And we can keep track of how this performs. @shannonbradshaw made a good point of finding a term that captures both DR + HA, as often "Resilience" only tends to stand in for one of the other.